recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/doma-salvator-xs-h3.cfg

```diff
@@ -47,7 +47,7 @@ vif = [ 'backend=DomD,bridge=xenbr0,mac=08:00:27:ff:cb:ce' ]
 # connector 1001 - second android connector on AGL application display (1088x1487):
 #     * AGL provides layout (x=0, y=218, w=1080, h=1487) for HMI application;
 #     * we use 1088 to be aligned with tile 32 (pixels), it will be scaled to 1080 by DisplayManager.
-vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:768x1024' ]
+vdispl = [ 'backend=DomD,be-alloc=0,connectors=1000:1920x1080;1001:1024x768' ]
```

Comment:
What about other config files? doma-salvator-x-h3-4x2g.cfg for example.

Reply:
Also I think that you should to correct comment above "vdispl =" string. Seems it is not relevant.

Reply:
Updated

Reply:
But the comment above the vdispl still does refer AGL which gone away long ago. And describes outdated displays layout.